internal/records/record_builder.go

```diff
@@ -558,6 +561,18 @@ func newFieldBuild(dt arrow.DataType, mem memory.Allocator, name string, nullabl
 			e.Append(v.Int())
 			return nil
 		}
+	case *array.Uint64Builder:
```

Comment:
If we support `*array.Uint64Builder` we also need to support `*array.Uint64DictionaryBuilder` . And I suggest the switch case for `*array.Uint64Builder` to be after `*array.Int64DictionaryBuilder`.

Also you need to add test case for `uint64` in  `record_builder_test.go` to make sure everything works as intended.

Reply:
Thanks for your suggestion, raised some commit for them. Please review it again when you have time.

Reply:
Besides, if I want to implement the arrow builders for other primitive types, what should I do after I add the enum values in the proto files?